script.js

```diff
@@ -21,3 +23,87 @@ window.onload = () => {
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: ['https://tufoinnkuo10.github.io/Personal-Portfolio', 'https://github.com/tufoinnkuo10/Personal-Portfolio'],
+  },
+  {
+    Image: './image/dash-data.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    name: 'card 3',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: ['https://tufoinnkuo10.github.io/Personal-Portfolio', 'https://github.com/tufoinnkuo10/Personal-Portfolio'],
+  },
+  {
+    Image: './image/dash-data.png',
+    description: 'A daily selection of privatel personalized reads; no accountssign-ups required. Has been the industry standard',
+    name: 'card 4',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: ['https://tufoinnkuo10.github.io/Personal-Portfolio', 'https://github.com/tufoinnkuo10/Personal-Portfolio'],
+  },
+  {
+    Image: './image/websitepro.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    name: 'card 5',
+    technologies: ['Python', 'React', 'bootstrap'],
+    links: ['https://tufoinnkuo10.github.io/Personal-Portfolio', 'https://github.com/tufoinnkuo10/Personal-Portfolio'],
+  },
+  {
+    Image: './image/websitepro.png',
+    description: 'A daily selection of privatel personalized reads; no accountssign-ups required. Has been the industry standard',
+    name: 'card 6',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    links: ['https://tufoinnkuo10.github.io/Personal-Portfolio', 'https://github.com/tufoinnkuo10/Personal-Portfolio'],
+  },
+];
+
+const buttons = document.querySelectorAll('.cproject');
+buttons.forEach((btn, i) => {
+  btn.addEventListener('click', () => {
+    const project = projectContent[i];
+
+    const popup = document.querySelector('.pop-up');
+    popup.style.display = 'block';
+
+    // get title of popup
+    const popupTitle = popup.querySelector('.title');
+    const popupImage = popup.querySelector('.pop-img');
+    const popupTech = popup.querySelector('.btns');
+    const popupDesc = popup.querySelector('.pop-text2');
+
```

Comment:
- [ ] The modal pop up should be in the center for both mobile view and desktop view as per Figma design. Your modal is aligned to left in desktop view and is taking the whole width in mobile view. Kindly try to implement the modal as per Figma's design.

- [ ] The background colour of buttons is white. Kindly try to change the colour technology button's colour from grey to white.

- [ ]  On desktop view there is less margin between the projected image and description. Kindly check the below comparison and fix it.

- [ ] The icon on the source button should be aligned properly to the text. Kindly check the below design and fix it.

Your modal on Desktop view
![image](https://user-images.githubusercontent.com/53019916/131695441-a202589a-6cef-4919-9eed-1872d5e9e9cd.png)

Figma design for desktop view
![image](https://user-images.githubusercontent.com/53019916/131695575-bb59282c-7d30-41c6-bb99-9fa69695023d.png)

Your modal pop up on mobile view
![image](https://user-images.githubusercontent.com/53019916/131695796-7606ada6-249f-41f3-9165-833fbda10498.png)

Figma design for mobile view
![image](https://user-images.githubusercontent.com/53019916/131695912-4ed6088f-f042-43ac-b532-33387af2589d.png)